DbcParserLib/Packer.cs

```diff
@@ -99,14 +103,14 @@ public static double RxSignalUnpack(ulong RxMsg64, Signal signal)
         /// <param name="receiveMessage">The message data</param>
         /// <param name="signal">Signal containing dbc information</param>
         /// <returns>Returns a double value representing the unpacked signal</returns>
-        public static double RxSignalUnpack(byte[] receiveMessage, Signal signal)
+        public static double RxSignalUnpack(uint8_T[] receiveMessage, Signal signal)
         {
             var startBit = signal.StartBit;
             var message = receiveMessage;
 
             if (!signal.Intel())
             {
-                var tempArray = new byte[message.Length];
+                var tempArray = new uint8_T[message.Length];
                 Array.Copy(message, tempArray, message.Length);
```

Comment:
This part is bittersweet. It's easy and readable but allocating to just read backward it's a pity. 
Maybe if we change the `ExtractBits` signature into something like
```
private static uint64_T ExtractBits(uint8_T[] data, int startBit, int endBit, int step = 1) // Or -1 if backward
```
we could keep the same iteration without assuming that length is the limit. Just food for thoughts.

...or... (this is a bit outside the scope of this PR, but worth mentioning)
Today the `Packer` is a static class so it needs Signal and message info. If we optimize the `Signal` class (or if we adda a sort of `.GetReader()` on it) then we could optimize some stuff, removing ifs, and give a forward/backward reader depending on endianness